tests/test_api_auth.py

```diff
@@ -312,10 +299,6 @@ def test_02_auth_without_split(self):
                 self.assertEqual(401, res.status_code, res)
                 result = res.json.get("result")
                 self.assertFalse(result.get("status"), result)
-            if six.PY2:
-                expected = "The user User(login=u'cornelius@unknown', " \
-                           "realm=u'realm1', resolver='') exists in NO resolver."
-            else:
                 expected = "The user User(login='cornelius@unknown', " \
```

Comment:
here also

Reply:
fixed in a1e752d6c5abb37fab930e65b3c64b3680e3f3f1.